rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/requester.hpp

```diff
@@ -137,10 +139,8 @@ class Requester
       estr = "DomainParticipant::create_topic: failed for response";
       goto fail;
     }
-
-    // Let OpenSplice do any needed encoding
     content_filtered_response_topic_ = participant_->create_contentfilteredtopic(
-      service_name_.c_str(), response_topic_,
+      content_filtered_topic_name.c_str(), response_topic_,
```

Comment:
I just checked with `rtiddspy` and you are right, Connext creates a `ContentFilteredTopic` whose topic name is different per requester.